third_party/llvm_openmp/openmp.bzl

```
@@ -19,3 +30,72 @@ def dict_add(*dictionaries):
     for d in dictionaries:
         result.update(d)
     return result
+
+def select_os_specific(L, M, W):
+    return select({
+        "@org_tensorflow//tensorflow:linux_x86_64": L,
+        "@org_tensorflow//tensorflow:macos": M,
+        "@org_tensorflow//tensorflow:windows": W,
+        "//conditions:default": L,
+    })
+
+def libname_os_specific():
+    return "" + select_os_specific(L = "libiomp5.so", M = "libiomp5.dylib", W = "libiomp5md.dll")
```

Nit: We can refactor "libiomp5" out.
```suggestion
    return "libiomp5" + select_os_specific(L = ".so", M = ".dylib", W = ".dll")
```